feature/google/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -684,7 +684,13 @@ class BillingWrapperTest {
             mockClient.queryPurchases(any())
         } returns Purchase.PurchasesResult(BillingClient.BillingResponseCode.OK.buildResult(), null)
 
-        assertThat(wrapper.queryPurchases(BillingClient.SkuType.SUBS)!!.purchasesByHashedToken).isNotNull
+        var result: BillingAbstract.QueryPurchasesResult? = null
+        wrapper.queryPurchases("appUserID") {
+            result = it
+        }
+
+        assertThat(result).isNotNull
```

Comment:
the name of the method says "returns an empty list", but that's not what we're checking here, perhaps we could add a check for that? 
otherwise, the test could still pass if this returns something else, as long as it isn't null